eigen.sh

```diff
@@ -0,0 +1,45 @@
+package: Eigen
+version:  "3.3.4"
+build_requires:
+  - CMake
+---
+#/bin/sh
+# download package
+URL="https://bitbucket.org/eigen/eigen/get/3.3.4.tar.bz2"
+file="eigen.tar.bz2"
+#download if newer than local copy
+if test -e "$file"
+then
+  #md5sum $file > $file".md5"
+  curl -Lo $file "-z '$file'" "$URL"
+else
+  curl -Lo $file "$URL"
+  #md5sum $file > $file".md5"
+fi
+# extract, but do'nt overwrite existing files
+#if md5sum $file -c $file".md5"
+#then
+tar -xjf $file -C $SOURCEDIR --strip-components=1
+#fi
```

Comment:
Lines 7-23 are working around aliBuild, you need to specify a git repo instead. In the README file in the alfadist repo you find a section on what to do when there is no git repo yet. Pls reference a git repo and remove lines 7-23. Thx!